src/flags/blocks.rs

```diff
@@ -153,10 +162,16 @@ impl Configurable<Self> for Blocks {
             } else {
                 Some(Self(blocks))
             }
+        } else if let Some(true) = config.always_long {
+            Some(Self::long())
         } else {
             None
         }
     }
+
+    fn from_environment() -> Option<Self> {
```

Comment:
Whoops! Totally forgot I had added that. I think at one point I was experimenting with using the environment to override the config option rather than a CLI flag.